CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(smartsendercpp)
 
 set(CMAKE_CXX_STANDARD 20)
 
-add_executable(smartsendercpp "src/CMakeLists.txt" "src/models/CMakeLists.txt" "src/utils/CMakeLists.txt")
+add_executable(smartsendercpp "src/CMakeLists.txt" "src/models/CMakeLists.txt" "src/utils/CMakeLists.txt" src/models/TextCompressor.cpp src/models/TextCompressor.h)
```

Comment:
skal tilføjes til CMakeLists.txt filen inde i models-mappen i stedet for

Reply:
Fixed.